bonus/bonusAvailable.md

```diff
@@ -15,6 +15,9 @@
 
 ### bonus_available_v2.json
 - pointed by app > `1.10.0.4`
+- A new logic for handling a bonus visibility has been implemented. Now every bonus has a new property `visibility` which can be evaluated with 3 different values:
```

Comment:
```suggestion
- '> 1.18.0.3' A new logic for handling a bonus visibility has been implemented. Now every bonus has a new property `visibility` which can be evaluated with 3 different values:
```
What do you think if we write the last version that implement the old logic?
in this way we can avoid **WIP** in other points

Reply:
I was actually waiting for the latest version of the app to be released 😄 now I can update readme information!